data/MuhammadRabi.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "Muhammad Rabi",
+  "type": "personal",
+  "bio": "I'm an Autodidact Front-end web developer, seeking contribution to open source projects",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/MuhammadRabi",
+      "icon": "FaGitHub"
+    },
+    {
+      "name": "Follow me on Linkedin",
+      "url": "https://www.linkedin.com/in/muhammad-rabi-07a51b202/",
+      "icon": "FaLinkedIn"
```

Same here. It should be `FaLinkedin`

```suggestion
      "icon": "FaLinkedin"
```

---

Thank you for your help